addons/token-exchange/blue_secret_controller.go

```diff
@@ -125,6 +140,51 @@ func (c *blueSecretTokenExchangeAgentController) sync(ctx context.Context, syncC
 	return nil
 }
 
+func (c *blueSecretTokenExchangeAgentController) enableMirroring(storageClusterName string) error {
+	ctx := context.TODO()
```

Comment:
Let it take the context of caller function as argument.

Reply:
done